### ISSUE TITLE
[BUG] eSim doesn't receive calls with VoLTE on

### ISSUE THREAD

**Issue by elaouby** [Lethany/BeyondROM_Hub#5]
**Describe the bug**
G986B
Sim Tray:
SIM 1: is an international roaming sim that is just being monitored
SIM 2: Empty
SD: Sandisk 512GB
eSIM: eSIM installed and used as the main sim for calls and data

Phone can't receive calls when VoLTE is turned on for eSim.

When VoLTE is on:
Data working, outgoing calls are working, incoming calls automatically turn off as soon as call is answered

When VoLTE is off:
Data working, outgoing calls are working, incoming calls working

Tested with original flashed firmware, problem doesn't happen

**Comment by Lethany** [comment]
@elaouby Where is your provided logcat that you are mean't to grab when the bug happens? use log-e.log

**Comment by elaouby** [comment]
[log_e.log](https://github.com/Lethany/BeyondROM_Hub/files/7667952/log_e.log)

Attached below, apologies.

**Comment by Lethany** [comment]
@elaouby What firmware are you running?

**Comment by elaouby** [comment]
Currently on BR3.9 DUJ5, problem was also there in 3.6, 3.7, 3.8
Not sure if it was there before 3.6 because I only got the eSIM in 3.6

**Comment by Lethany** [comment]
@elaouby You used CSC mods?

**Comment by elaouby** [comment]
Yes, all CSC mods were ticket in Aroma setup except for Knoxguard

**Comment by Lethany** [comment]
@elaouby eSim option shows in it's usual place in settings?

**Comment by Lethany** [comment]
@elaouby Can you use log-v.log when making and receiving a call on the eSIM with VoLTE on

**Comment by LeeXDA21** [comment]
@elaouby I am closing this issue due to no further responses from you. Thank you for reporting this as a bug, if this bug still exists then please open another issue. Thanks.
